packages/next-steps/index.tsx

```diff
@@ -61,7 +61,31 @@ export default function NextSteps({
                       <p className={s.tileDescription}>{step.description}</p>
                     ) : null}
                     <span className={s.tileCta}>
-                      {step.cta.title} <IconArrowRight16 />
+                      <span className={s.tileCtaLabel}>{step.cta.title}</span>
```

Comment:
💭 Non-blocking: looks like we're copying this over from `StandaloneLink`. Any thoughts on abstracting this or parts of this out somewhere?

Reply:
The tricky part when thinking about abstracting this is the parent element that is triggering the hover animation. In StandaloneLink it is the link, in NextSteps it's the tile. Not to say its not possible but it felt ok to copy paste this for now.

Reply:
_Oh interestin'_. That makes sense!